src/lib/sort/sort-header.ts

```diff
@@ -50,12 +58,14 @@ const SORT_ANIMATION_TRANSITION =
   templateUrl: 'sort-header.html',
   styleUrls: ['sort-header.css'],
   host: {
-    '(click)': '_sort.sort(this)',
+    '(click)': '_handleClick()',
     '[class.mat-sort-header-sorted]': '_isSorted()',
+    '[class.mat-sort-header-disabled]': '_isDisabled()',
   },
   encapsulation: ViewEncapsulation.None,
   preserveWhitespaces: false,
   changeDetection: ChangeDetectionStrategy.OnPush,
+  inputs: ['disabled: matSortHeaderDisabled'],
```

Comment:
Should be `disabled`

Reply:
I didn't go with `disabled`, because while it is a component, the select is still an attribute so it's not super obvious what you're disabling.

Reply:
Yeah, just a funny consequence of our rules - might loop in @jelbourn for confirmation on this but as a rule, components don't prefix

Reply:
Switched it to `disabled` @andrewseguin.